meta-refkit/conf/distro/include/refkit-development.inc

```diff
@@ -4,7 +4,7 @@ IMA_EVM_KEY_DIR = "${IMA_EVM_BASE}/data/debug-keys"
 # Enable local root access. Always use _append, to
 # avoid getting this change overwritten by a
 # REFKIT_IMAGE_EXTRA_FEATURES = "foo" in local.conf.
-REFKIT_IMAGE_EXTRA_FEATURES_append = " autologin"
+REFKIT_IMAGE_EXTRA_FEATURES_append = " autologin debug-tweaks"
```

Comment:
With real hardware we first flash the image to emmc/sd card and then add the ssh-key to authorized_keys but it's done by DAFT program that isn't used with QEMU. So I thought this might be the simplest solution but I understand why it couldn't be used. So I guess I'll try to add ssh-key with serial console.